src/main/java/xyz/vegaone/easytracking/service/ProjectService.java

```diff
@@ -19,21 +19,26 @@
 @Service
 public class ProjectService {
 
```

Comment:
There are some unused imports in this class and some commented code at line 53, could you please remove them?